nautobot/dcim/templates/dcim/powerfeed.html

```diff
@@ -62,7 +62,7 @@ <h1>{% block title %}{{ object }}{% endblock %}</h1>
 
 {% block content %}
 <div class="row">
-	<div class="col-md-6">
+    <div class="col-md-6">
```

Comment:
I'd like to keep this indent change as it goes to on the line that should be used.